contracts/upgradeable_contracts/erc20_to_native/FeeManagerErcToNativePOSDAO.sol

```diff
@@ -1,36 +1,20 @@
 pragma solidity 0.4.24;
 
-import "../BaseFeeManager.sol";
 import "../../IBlockReward.sol";
+import "../BlockRewardFeeManager.sol";
 
-contract FeeManagerErcToNativePOSDAO is BaseFeeManager {
+contract FeeManagerErcToNativePOSDAO is BlockRewardFeeManager {
```

Comment:
@akolotov please check https://github.com/poanetwork/poa-bridge-contracts/pull/164/commits/04069bbb7e20ac13c7a1159ccd8ad22b684626fd. Updated `FeeManagerErcToNativePOSDAO` to extend from `BlockRewardFeeManager`

Reply:
Great! It looks OK for me